DESCRIPTION

```diff
@@ -27,11 +27,12 @@ Imports:
     methods,
     utils
 Suggests:
-    covr
+    covr,
+    roxut
 URL: 
     https://r-hyperspec.github.io/hySpc.read.jdx,
     https://github.com/r-hyperspec/hySpc.read.jdx
 BugReports: https://github.com/r-hyperspec/hySpc.read.jdx/issues
 RoxygenNote: 7.1.1
-Roxygen: list(markdown = TRUE)
+Roxygen: list(markdown = TRUE, packages = "roxut", roclets = c("namespace", "rd", "roxut::tests_roclet"))
```

Comment:
in general (thinking of the package skeleton), we'd add also the `"collate"` and `"vignette"` roclets here, right?

Side note: see https://github.com/rstudio/rstudio/issues/6433#issuecomment-667847661 for working in RStudio

Reply:
Yes, I just copied the string from another package that uses rouxt so we should refine what `r-hyperspec/...` needs.